test/test_functional.py

```diff
@@ -2,13 +2,13 @@
 from IO layer as well as UI later """
 import os
 import unittest
-from tkinter import Tk
+# from tkinter import Tk
 import time
 import subprocess
 from test.test_resource import TestResource
 import pandas as pd
 from similarity_processor.similarity_io import SimilarityIO
-from similarity_processor.similarity_ui import TextSimilarityWindow
+# from similarity_processor.similarity_ui import TextSimilarityWindow
```

Comment:
Should we remove the commented code?

Reply:
They are UI test which can be done at desk, but CI it is not possible hence kept as commented. In travis there was a skip option. such option i am not able to get in github action. Hence keeping it commented untill i find a way to skip the test in CI